.rubocop.yml

```diff
@@ -0,0 +1,197 @@
+AllCops:
+  DisplayCopNames: true
+  TargetRubyVersion: 2.3
```

Comment:
I don't remember the details, but using 2.3 might cause Rubocop to enforce some rules that are not compatible with 2.2.